super_csv/csv_processor.py

```diff
@@ -218,7 +218,7 @@ def preprocess_file(self, reader):
             result = ResultDict(row)
             try:
                 self.validate_row(row)
-                row = self.preprocess_row(row)
```

Comment:
This isn't backwards compatible. I believe that we use super csv in a few places other than bulk_grades, and this will break them. I'm actually gonna have to go brush up on my python inheritance before I can for sure say what we need to do, but it's not this simple